Dockerfile

```diff
@@ -22,7 +22,7 @@ COPY ./paddle/scripts/docker/root/ /root/
 
 RUN apt-get update && \
     apt-get install -y \
-    git python-pip python-dev openssh-server bison libnccl-dev \
```

Comment:
Should also update `tools/manylinux1/Dockerfile.x64` for release builds.

Reply:
`libnccl` is not at the `nvidia/cuda:8.0-cudnn5-devel-ubuntu16.04`. So I added it back. In this case, we don't need to update `tools/manylinux1/Dockerfile.x64`. :)